### DISCUSSION TITLE
Parsing PDF documents with paths instead of text

### DISCUSSION THREAD

**Discussion by jdales** [UglyToad/PdfPig#733] [Q&A]
We've been using PdfPig to process PDF forms and, until just recently, had great success extracting text from documents.  We've run into documents that contain pages, but no letters and result in no text extraction.  The operations of our successful extractions contain a series of Text operations (e.g. TextObjects.BeginText, TextObjects.SetFontAndSize, TextObjects.ShowText, etc...), but these new documents contain PathConstruction operations (e.g. PathConstruciton.AppendStraightLineSegment, PathConstruction.BeginNewSubpath, etc...). Is there a way for us to extract the text from these new documents?

**Comment by topcat30** [comment]
maybe the hocr would work
